lmdeploy/pytorch_poc/engine/engine.py

```diff
@@ -152,9 +153,12 @@ def __init__(self,
                  model_path: str,
                  scheduler_config: SchedulerConfig = None,
                  cache_config: CacheConfig = None) -> None:
-        hf_model = AutoModelForCausalLM.from_pretrained(model_path)
-        self.patched_model = patch(hf_model, ['context']).cuda()
+        with LoadNoInit():
```

Comment:
what's the differents between this and `from accelerate import init_empty_weights`